.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Set environment variables that depend on other environment variables.
+    - run: |
+        echo ::set-env name=WIT_PACKAGE_UNDER_TEST_PATH::"$GITHUB_WORKSPACE/wit-packages/api-scala-sifive.git"
+        echo ::set-env name=WIT_WORKSPACE::"$GITHUB_WORKSPACE/wit-workspace"
+
+    - uses: actions/checkout@v2
+      with:
+        path: ${{ env.WIT_PACKAGE_UNDER_TEST_PATH }}
+
+    - name: Wit init
+      run: |
+        mkdir -p "$WIT_WORKSPACE"
+        docker run \
+          --volume="$WIT_PACKAGE_UNDER_TEST_PATH:/mnt/api-scala-sifive.git" \
+          --volume="$WIT_WORKSPACE:/mnt/workspace" \
+          --workdir="/mnt/workspace" \
+          --rm \
+          sifive/wit:v0.11.1 \
+          bash -c "git config --global url.'https://github.com/'.insteadOf 'git@github.com:' && wit -vvvv init . -a /mnt/api-scala-sifive.git -a git@github.com:sifive/environment-blockci-sifive.git::0.3.0"
```

Comment:
Do you really want to run `wit` with `-vvvv`? I suspect you were doing that for debugging but I don't think it's what we want committed

Reply:
Ah, I missed this while I was cleaning up my debugging commits. This line is pretty gnarly and hard to read because it's several layers of strings embedded in different languages. I'll remove it.